lxd/main.go

```diff
@@ -947,6 +953,18 @@ they otherwise would.
 		}
 	}
 
+	if imagesAutoUpdate {
+		_, err = c.SetServerConfig("images.auto_update_cached", "true")
```

Comment:
I think you could shorten all this by using fmt.Sprintf("%s", imagesAutoUpdate) if you wanted.